common/app/layout/FaciaContainer.scala

```diff
@@ -165,7 +166,7 @@ object FaciaContainer {
         case _            => ContainerCommercialOptions(omitMPU = false, adFree = adFree)
       },
       config.config.description.map(DescriptionMetaHeader),
-      None,
+      customClasses = config.config.metadata.flatMap(paletteClass(container, _)).map(Seq(_)),
```

Comment:
customClasses has a type of `Seq[String]`, but we only add one class String for palettes